docs/model-monitoring/applications.md

```diff
@@ -0,0 +1,114 @@
+# Writing a Model Monitoring Application
+
+## Basics
+
+First, create a Python module and import the API objects:
+
+```py
+from mlrun.model_monitoring.applications import (
+    ModelMonitoringApplicationBase,
+    ModelMonitoringApplicationMetric,
+    ModelMonitoringApplicationResult,
+    MonitoringApplicationContext,
+)
+from mlrun.common.schemas.model_monitoring import ResultKindApp, ResultStatusApp
+```
+
+Then, write the application itself by inheriting from the `ModelMonitoringApplicationBase` base class.
+You have to implement the `do_tracking` method.
+Here is a "dummy" app that returns a constant result for each monitoring window:
+
+```py
+class ServingMonitoring(ModelMonitoringApplicationBase):
+    def do_tracking(
+        self, context: MonitoringApplicationContext
+    ) -> ModelMonitoringApplicationResult:
+        return ModelMonitoringApplicationResult(
+            name="dummy-res-just-for-demo",
+            value=0,
+            status=ResultStatusApp.irrelevant,
+            kind=ResultKindApp.mm_app_anomaly,
+        )
+```
+
+The `do_tracking` method of the application is called for each monitoring window, and returns a list
+or results `ModelMonitoringApplicationResult` and metrics `ModelMonitoringApplicationMetric`, or just one
+result, as in the example above.
+
+```{versionchanged} 1.7.0 ModelMonitoringApplicationBase. Please update your application accordingly.
+```
+
```

Comment:
for completeness, please point to [registration section](file:///Users/Assaf_Ben_Amitai/iguazio/misc_ws/mlrun/docs/_build/html/model-monitoring/model-monitoring-flow.html#register-and-deploy-the-model-monitoring-app)